hicexplorer/hicFindTADs.py

```diff
@@ -1375,7 +1382,7 @@ def main(args=None):
     elif not os.path.isfile(tad_score_file):
         ft.compute_spectra_matrix()
         # save z-score matrix that is needed for find TADs algorithm
-        ft.hic_ma.save(args.outPrefix + "_zscore_matrix.h5")
+        ft.hic_ma.save(args.outPrefix + "_zscore_matrix." + matrix_ending)
```

Comment:
You could use:
`ft.hic_ma.save(zscore_matrix_file)`

Reply:
yes, this can be changed. I put it on my TODO list.